docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -465,10 +465,10 @@ default. The procedure is enabled only when
 `delta.register-table-procedure.enabled` is set to `true`.
 
 (delta-lake-unregister-table)=
-
 #### Unregister table
 
-The connector can unregister existing Delta Lake tables from the metastore.
+The connector can unregister existing Delta Lake tables from the metastore, if
```

Comment:
to be confirmed

Reply:
Whoa .. so registering a table is gated by a property that defaults to false and unregistering is not gated a all. Thats super weird.. I certainly dont understand the logic of that. @martint was already wondering as well .. do we need to revisit the code for this .. also wdyt @electrum ?

Reply:
`unregister_table` procedure internally calls ConnectorAccessControl as I replied in Slack. Note that the procedure is almost same as DROP TABLE statement (the only difference is whether removing data or not).

Reply:
Okay .. I updated the PR .. if we change code we can update separately. More discussion on slack.